src/components/index/demo-display.tsx

```diff
@@ -24,8 +25,20 @@ const DemoDisplay = ({demo, descript}: DemoDisplayProps ) => {
         }
     }, [demo, descript]);
 
+    let additonalStyle:string = '';
+    
+    if (currentTab === 0) {
+        additonalStyle = '0px 10px 10px 10px';
+    }
+    else if (currentTab === 3) {
```

Comment:
Change the 3 to be the length of the tabs instead of being hard coded since we might add more later. They will be coming from a collection in Directus, so you should just be able to check the length.